src/tests/e2e/recipients.test.e2e.js

```diff
@@ -0,0 +1,96 @@
+import LoginPage from './pages/login.page'
+import RecipientsPage from './pages/recipients.page'
+import paths from '../../Paths'
+import log from 'loglevel'
+log.setDefaultLevel('info')
+
+const timeout = 50000
+
+describe('Recipients Page', () => {
+  let loginPage
+  let recipientsPage
+  beforeAll(async done => {
+    // we assume user has not logged in yet
+    await page.goto(process.env.E2E_TEST_URL)
+    loginPage = new LoginPage()
+    recipientsPage = new RecipientsPage()
+    await loginPage.login(
+      process.env.E2E_TEST_GOOGLE_LOGIN_USERNAME,
+      process.env.E2E_TEST_GOOGLE_LOGIN_PASSWORD,
+      true // mock login
+    )
+    // we should not see any error snackbars
+    await recipientsPage.navigateFromLandingPage()
```

Comment:
This should not be done by recipientsPage, it should assume it's already in /contacts